lib/actions/rttDevice.js

```diff
@@ -123,11 +143,14 @@ class RTTDevice {
                 this.isRttOpen = true;
             })
             .then(() => RTTDevice.getHardwareStates())
-            .then(({ string }) => {
+            .then(({ string, iteration }) => {
+                console.log(`it took ${iteration} iteration to read hw states`);
```

Comment:
Suggest we remove the `console.log`

Reply:
I thought about that, but I find it useful. Nothing suggests that the connection will be made in 250 iterations, and in that case it will be valuable information why the connection wasn't made.

Reply:
I agree, since this is something that is only used during startup.